src/DssSpell.sol

```diff
@@ -60,222 +58,171 @@ contract DssSpellAction is DssAction {
 
-        // add the new ClipperMomJob (0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF) to the Cron Sequencer
-        DssCronSequencerLike(CRON_SEQUENCER).addJob(CRON_CLIPPER_MOM_JOB_NEW);
+        // ----- Update PSM-USDC-A DC-IAM -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // Update CRON_CLIPPER_MOM_JOB to 0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF in the Chainlog
-        DssExecLib.setChangelogAddress("CRON_CLIPPER_MOM_JOB", CRON_CLIPPER_MOM_JOB_NEW);
+        // ----- Update MCD_LITE_PSM_USDC_A Buf -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // Note: bump chainlog version due to the updated CRON_CLIPPER_MOM_JOB address
-        DssExecLib.setChangelogVersion("1.17.6");
+        // ----- Update LITE-PSM-USDC-A DC-IAM -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // ---------- Aligned Delegate MKR Compensation ----------
-        // Forum: https://forum.makerdao.com/t/july-2024-aligned-delegate-payment-requests/24794
-        // MIP: https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-budget-and-participation-requirements
+        // Note: load the MCD contracts depencencies
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
 
-        // BLUE - 13.75 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 13.75 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Note: specify the migration config
+        DssLitePsmMigrationConfigPhase2 memory cfg = DssLitePsmMigrationConfigPhase2({
+            // Note: chainlog key of new psm lite
+            dstPsmKey:  "MCD_LITE_PSM_USDC_A",
 
-        // Cloaky - 12.00 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 12.00 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // MCD_LITE_PSM_USDC_A buf: Increase by 180 million DAI, from 20 million DAI to 200 million DAI
+            dstBuf:     200 * MILLION * WAD,
 
-        // Byteron - 1.25 MKR - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        MKR.transfer(BYTERON, 1.25 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // Increase by 7,450 million DAI, from 50 million DAI to 7,500 million DAI.
+            dstMaxLine: 7_500 * MILLION * RAD,
 
-        // JuliaChang - 1.25 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 1.25 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // Increase by 180 million DAI, from 20 million DAI to 200 million DAI.
```

Comment:
Please update those 2 instructions to exactly match the Exec Sheet https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=175785542#gid=175785542&range=B58:B59

Reply:
Fixed in [5f41aea](https://github.com/makerdao/spells-mainnet/pull/422/commits/5f41aea961e1e15d115cd7d03108b27f02bdee7d)